frontend/src/metabase/query_builder/components/DataSelector/DataSelector.jsx

```diff
@@ -123,6 +121,18 @@ export function FieldDataSelector(props) {
   );
 }
 
+export function RawDataBackButton() {
```

Comment:
nit: It's not used here, can we move it to a a separate file?

Reply:
done